src/main/src/test/java/org/geoserver/security/xml/XMLUserDetailsServiceTest.java

```diff
@@ -44,6 +37,11 @@ protected SystemTestData createTestData() throws Exception {
     }
 
     @Override
+    protected void setUpTestData(SystemTestData testData) throws Exception {
+        // do nothing here, we want the live test data info and run its migration
```

Comment:
This is the one case in which for some reason we need migrations to run, or the test will fail.